components/sidebar/index.js

```diff
@@ -26,6 +26,20 @@ import {getIsLhsOpen} from 'selectors/lhs';
 
 import Sidebar from './sidebar.jsx';
 
+const getTeamUnreads = createSelector(
```

Comment:
I find the name confusing, as I understand this sums up the counters across all user's teams. I'd suggest maybe `getTotalUnreads`? Better yet, maybe reuse the old simple name `getUnreads`, different in this context? That may also be confusing though.

Reply:
I totally agree.  I updated it to `getTotalUnreads`. Thanks @levb!